ci-operator/step-registry/openshift/e2e/gcp/windows-node/openshift-e2e-gcp-windows-node-workflow.yaml

```diff
@@ -0,0 +1,27 @@
+workflow:
+  as: openshift-e2e-gcp-windows-node
+  steps:
+    env:
+      SUB_CHANNEL: stable
+      SUB_INSTALL_NAMESPACE: openshift-windows-machine-config-operator
+      SUB_PACKAGE: windows-machine-config-operator
+      SUB_TARGET_NAMESPACES: '!install'
+    pre:
+      - chain: ipi-conf-gcp
+      - ref: ovn-conf
+      - ref: ovn-conf-hybrid-manifest
+      - chain: ipi-install
+      - ref: operatorhub-subscribe
+      - ref: windows-conf-operator
+      - ref: ipi-conf-gcp-windows-machineset
+    test:
+      - ref: openshift-e2e-test
```

Comment:
This is the [general OCP payload test suite](https://steps.ci.openshift.org/reference/openshift-e2e-test), not related to Windows
The WinC test step is named [windows-e2e-operator-test](https://steps.ci.openshift.org/reference/windows-e2e-operator-test)